conf/config-schema.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * @fileoverview Defines a schema for configs.
+ * @author Sylvan Mably
+ */
+
+"use strict";
+
+const baseConfigProperties = {
+    env: { type: "object" },
+    globals: { type: "object" },
+    parser: { type: ["string", "null"] },
+    parserOptions: { type: "object" },
+    plugins: { type: "array" },
+    rules: { type: "object" },
+    settings: { type: "object" }
+};
+
+const globList = {
+    oneOf: [
+        { type: "string" },
+        {
+            type: "array",
+            items: { type: "string" },
+            minItems: 1
+        }
+    ]
+};
+
+const overrideProperties = Object.assign(
+    {},
+    baseConfigProperties,
+    {
+        files: globList,
+        excludedFiles: globList
```

Comment:
This isn't a blocker, but it seems like it might be useful to allow empty excludedFiles arrays -- I know some people like to explicitly state default options in their config (e.g. in case they want to exclude files later, they won't have to look up the name again). At the moment, `excludedFiles` has `minItems: 1`.